.github/workflows/operator-integration-test.yml

```diff
@@ -53,12 +53,14 @@ jobs:
       - name: Deploy operator to minikube
         run: |
           make deploy
+          kubectl wait --for=condition=Ready pod --all -n amazon-cloudwatch
```

Comment:
this is the main reason why it was flaky. operator wasn't ready to serve on the webhook.